frontend/src/metabase/lib/settings.js

```diff
@@ -121,6 +121,11 @@ class Settings {
     return this.get("anon-tracking-enabled") || false;
   }
 
+  formattingOptions() {
+    const opts = this.get("custom-formatting");
+    return opts && opts["type/Temporal"] ? opts["type/Temporal"] : {};
```

Comment:
Sometimes I miss my optional chaining :)

Reply:
Oh yeah, I feel it 😄